templates/base.html

```diff
@@ -102,7 +102,7 @@
 
                     <meta property="schema:alternateName" content="Center for Digital Humanities @ Princeton University" />
                     <meta property="schema:alternateName" content="CDH @ Princeton University" />
-                    <meta property="schema:url" content="{% fullurl 'home' %}"/>
+                    <meta property="schema:url" content="https://{{ site.hostname }}{% pageurl site.root_page %}"/>
 
```

Comment:
wonder if it's worth also putting this in the global context processor? not sure where else it'd get used. it also just occurred to me you could try `site.root_page.get_full_url` instead of concatenating with the hostname

Reply:
update: tried the latter and it works! it gave me an http URL in local dev, but I think it's smart enough to use https in qa/prod because wagtail takes that into account when constructing URLs. but we can check.

Reply:
perfect! will revise, and glad to know about `get_full_url`